.changes/init.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+
```

Comment:
Let's add an echo or comment with an explanation of the purpose of this script?

Reply:
(we have a very similar script in go-vcloud-director/.changes/)

Reply:
Thanks!

> (we have a very similar script in go-vcloud-director/.changes/)

I wouldn't mind of such comment in the govcd too :)